jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -110,31 +122,105 @@
     @Override
     public void execute() throws MojoExecutionException {
         this.getLog().debug("Initializing Schema Generator");
-        final SchemaGenerator generator = this.createGenerator();
```

Comment:
This log message is wrong now because the generator is only initialized before it is used the first time (i.e. later on).
Why not keep the initialization here at the top? For simplicity's sake, we could just call `getGenerator()` here.

Reply:
I moved the debug log into the getGenerator() method. This makes it correct again.

The simplification I leave to you. It is a matter of taste and I do not care to much.
So you as owner decide what you want to have.